test/evergreen.yml

```diff
@@ -4266,8 +4266,8 @@ tasks:
             # Set the cyclomatic complexity limit to 20
             python3 "../metrixplusplus/metrix++.py" limit --max-limit=std.code.complexity:cyclomatic:20
 
-            # Fail if there are functions with cyclomatic complexity larger than 98
-            python "../metrixplusplus/metrix++.py" limit --max-limit=std.code.complexity:cyclomatic:98 > $t
+            # Fail if there are functions with cyclomatic complexity larger than 89
```

Comment:
(nit?) How about leaving some leeway above the current maximum? Say, 1.5 * max ?

Reply:
I think we want to know every time it is breached. If necessary, we can increase this in the change that breaches this.

Reply:
+1 
We start warning users about complex functions when they exceed a score of 20. If a PR increases complexity past 89 it should be a visible and intentional change.